source/js/ellipsis-elements.js

```diff
@@ -0,0 +1,162 @@
+var characterPerLine = require("./characters-per-line");
+var errorHandling = require("./error-handling");
+var ellipsis = require("html-ellipsis");
+
+characterPerLine.initFontConstant();
+
+var ellipseableObjects = [];
```

Comment:
we have an EllipsisUtil object, I think we should think about to move the whole ellipsis into its own object, so that also "ellipseableObjects" and "ellipsisConfig" is encapsulated from the rest.

Reply:
Done